codebuild/bin/install_sslyze.sh

```diff
@@ -13,15 +13,25 @@
 # permissions and limitations under the License.
 #
 
-set -ex
+set -e
+. codebuild/bin/s2n_setup_env.sh
 
-python3 -m pip install --user --upgrade pip setuptools
+aarch64_install() {
+        echo "sslyze has a dependency on nassl, which will not build on ARM."
+}
 
-# Version 3.0.0 introduces backwards incompatible changes in the JSON we parse.
-# If we upgrade, the json format changes, breaking either Travis OR Codebuild.
-python3 -m pip install --user "sslyze<3.0.0"
-
-sudo ln -s /root/.local/bin/sslyze /usr/bin/sslyze || true
-
-which sslyze
-sslyze --version
+case "$ARCH" in
+  "aarch64")
+        aarch64_install
+        exit 1
+        ;;
+  *)
+        python3 -m pip install --user --upgrade pip setuptools
+        # Version 3.0.0 introduces backwards incompatible changes in the JSON we parse.
+        # If we upgrade, the json format changes, breaking either Codebuild.
```

Comment:
Nit: "breaking either Codebuild"? I'd leave it as "either Travis or Codebuild" even though we're no longer on Travis, since this is an explanation of why something was done in the past.

But since we've moved off of Travis, can now unpin the sslze version?

Reply:
Converted line 31 into a TODO; the [integration test](https://github.com/awslabs/s2n/blob/main/tests/integration/s2n_sslyze_test.py#L87) just needs to be checked against the newer output.

Reply:
Can you create an issue for it too?